source/simulator/lateral_averaging.cc

```diff
@@ -712,52 +769,59 @@ namespace aspect
       {
         if (property_names[property_index] == "temperature")
           {
-            functors.push_back(std_cxx14::make_unique<FunctorDepthAverageField<dim>>
+            functors.push_back(std::make_unique<FunctorDepthAverageField<dim>>
```

Comment:
@tjhei reminded me that we can't do that yet :-( (See #4188 .)

Reply:
Okay, I removed that from the last commit.